src/extrinsic/common.rs

```diff
@@ -0,0 +1,40 @@
+/*
+   Copyright 2019 Supercomputing Systems AG
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+// Common types.
+
+use codec::{Decode, Encode};
+use node_template_runtime::RuntimeCall;
+use sp_runtime::AccountId32;
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug)]
+pub struct PayoutStakers {
+    pub validator_stash: AccountId32,
```

Comment:
Can you change this to a `GenericAddress`?

Reply:
Actually its throwing an error like  "value: InvalidHexString(InvalidStringLength)'" after changing to GenericAddress type.

Reply:
Okay. Let's leave as is for now, then. Thanks for trying!